react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion.jsx';
+
+const example = [
+  {
+    title: 'Hypertext Markup Language',
+    content: 'Hypertext Markup Language(HTML) is the standard markup language for creating web pages and web applications.With Cascading Style Sheets(CSS) and Javascript, it forms a triad of cornerstone technologies for the World Wide Web.'
+  },
```

Comment:
Instead of using an index to track each item you can add an unique `id` property to each topic